### PR TITLE
reformatted docstring for keras integration

### PR DESCRIPTION
**Pull request by ariG23498**
Description
-----------

Reformat the docstring for the `keras` integration.

Documentation in GitBook corresponding to this PR can be found [here](https://arig23498.gitbook.io/aritra-documentation/~/revisions/-MZRJ0hS8iDvvKW7SJd2/ref/python/integrations/keras/wandbcallback)

### CODE REVIEW

File: wandb/integration/keras/keras.py [hunk 1]
```diff
@@ -215,34 +215,37 @@ class WandbCallback(keras.callbacks.Callback):
...
-            callbacks=[WandbCallback()])
+        ```python
+        model.fit(X_train,
+            y_train,
+            validation_data=(X_test, y_test),
+            callbacks=[WandbCallback()]
+        )
         ```
 
     WandbCallback will automatically log history data from any
```
Comment (charlesfrye) [inline wandb/integration/keras/keras.py:226]: I would suggest adding backticks to all instances of `WandbCallback` in the docstring, plus any other place where we we're using variable names/values. For example, in `monitor`: `auto`, `min`, and `max` should all be code-formatted, `model.fit` in the description of `training_data`, etc.

**Review by charlesfrye — CHANGES_REQUESTED** [review]
Nice! Just some minor changes here. Based on the page you included, there are some bigger changes that need to be made at the docugen end, but I'll put those reviews on your PR to that repo.

File: wandb/integration/keras/keras.py (hunk 1 above)
Comment (ariG23498) [reply wandb/integration/keras/keras.py]: Done!

**Comment by charlesfrye** [comment]
@raubitsj this PR is ready to go but blocked on CircleCI for unrelated failures.

**Review by charlesfrye — APPROVED** [review]
LGTM!

**Pull request merged**